prisma/schema.prisma

```diff
@@ -25,6 +25,8 @@ model Users {
   microsoftProfileId String?
   emailVerified      Boolean       @default(false)
   isDeleted          Boolean       @default(false)
+  isNew              Boolean       @default(true)
```

Comment:
When will we set this to false in DB?
I was thinking, if just the `isOnboarded` key will work for us? 🤔

Reply:
@prakashchoudhary07 , can you also add a document around this?

Reply:
We can probably use `onboarding` as a JSON datatype, with the required states added to it